lms/static/scripts/frontend_apps/components/FilePickerApp.js

```diff
@@ -138,7 +138,9 @@ export default function FilePickerApp({ onSubmit }) {
         method="POST"
         onSubmit={onSubmit}
       >
-        <h1 className="FilePickerApp__heading">Assignment details</h1>
+        <h1 className="FilePickerApp__heading text-2xl font-bold">
```

Comment:
Headings are unstyled after reset. This gets us back to where we were within a pixel or so of font size (it looks just fine). There are only a few headings in the LMS frontend app. One day we'll get to typography overall and get this standardized...

Reply:
> Did you think about adding heading rules for the whole frontend app, at a more general level?

ONE DAY we will get a chance to address our typography, across all our apps. I dream of it. Until then, we can get things into shape to make that dreamlike future possible...